builder.js

```diff
@@ -885,6 +885,37 @@ function shareBuild() {
     }
 }
 
+function saveBuild() {
+    if (player_build) {
+        let savedBuilds = window.localStorage.getItem("builds") === null ? {} : JSON.parse(window.localStorage.getItem("builds"));
+        let saveName = document.getElementById("build-name").value;
+        if ((!Object.keys(savedBuilds).includes(saveName) || document.getElementById("saved-error").textContent !== "") && location.hash !== "") {
+            savedBuilds[saveName] = location.hash.replace("#", "");
+            window.localStorage.setItem("builds", JSON.stringify(savedBuilds));
+
+            document.getElementById("saved-error").textContent = "";
+            document.getElementById("saved-build").textContent = "Build saved";
+        } else {
+            if (location.hash === "")
+                document.getElementById("saved-error").textContent = "Empty build";
+            else
+                document.getElementById("saved-error").textContent = "Exists. Overwrite?";
+        }
+    }
+}
+
+function loadBuild() {
+    let savedBuilds = window.localStorage.getItem("builds") === null ? {} : JSON.parse(window.localStorage.getItem("builds"));
```

Comment:
as a more general comment, does `window.localStorage` use the same space as `indexedDB` storage?

Reply:
No. LocalStorage stores strings only, and is synchronous. I haven't found any issues in testing at least.